Makefile

```diff
@@ -9,68 +59,69 @@ list:
 #############################
 
 list-strategies:
-	sudo docker-compose exec server zenbot list-strategies $(ARGS)
+	docker-compose exec server zenbot list-strategies $(ARGS)
 
 list-selectors:
-	sudo docker-compose exec server zenbot list-selectors $(ARGS)
+	docker-compose exec server zenbot list-selectors $(ARGS)
 
 backfill:
-	sudo docker-compose exec server zenbot backfill $(ARGS)
+	docker-compose exec server zenbot backfill $(ARGS)
 
 sim:
-	sudo docker-compose exec server zenbot sim $(ARGS)
+	docker-compose exec server zenbot sim $(ARGS)
 
 trade:
-	sudo docker-compose exec server zenbot trade $(ARGS)
+	docker-compose exec server zenbot trade $(ARGS)
 
 paper:
-	sudo docker-compose exec server zenbot trade --paper $(ARGS)
+	docker-compose exec server zenbot trade --paper $(ARGS)
 
 balance:
-	sudo docker-compose exec server zenbot balance $(ARGS)
+	docker-compose exec server zenbot balance $(ARGS)
 
 buy:
-	sudo docker-compose exec server zenbot buy $(ARGS)
+	docker-compose exec server zenbot buy $(ARGS)
 
 sell:
-	sudo docker-compose exec server zenbot sell $(ARGS)
+	docker-compose exec server zenbot sell $(ARGS)
 
 zenbot:
-	sudo docker-compose exec server zenbot $(ARGS)
+	docker-compose exec server zenbot $(ARGS)
 
 #############################
 # Docker machine states
 #############################
 
 up:
-	sudo docker-compose up -d
+	$(SUDO) docker-compose --file=$(DC_CONFIG) up
 
 start:
-	sudo docker-compose start
+	docker-compose start
 
 stop:
-	sudo docker-compose stop
+	docker-compose stop
 
 state:
-	sudo docker-compose ps
+	docker-compose ps
 
 rebuild:
-	sudo docker-compose stop
-	sudo docker-compose pull
-	sudo docker-compose rm --force server
-	sudo docker-compose rm --force mongodb
-	sudo docker-compose rm --force adminmongo
-	sudo docker-compose build --no-cache
-	sudo docker-compose up -d --force-recreate
+	$(SUDO) docker-compose stop
+	$(SUDO) docker-compose pull
+	$(SUDO) docker-compose rm --force server
+	$(SUDO) docker-compose rm --force mongodb
+	-$(SUDO) docker-compose rm --force adminmongo
```

Comment:
`-`?

Reply:
Are you asking about line 112?
That makes the command optional and prevents the target from stopping if it fails. If the user is using adminmongo, the command executes, otherwise the container doesn't exist so the target will report that the command failed and move on.